src/providers/rust.rs

```diff
@@ -94,22 +94,44 @@ impl RustProvider {
                 write!(build_cmd, " --package {workspace} --target {target}")?;
 
                 build.add_cmd(build_cmd);
+<<<<<<< HEAD
                 build.add_cmd(format!("cp target/{target}/release/{workspace} bin"));
             } else {
                 write!(build_cmd, " --target {target}")?;
 
                 if let Some(name) = RustProvider::get_app_name(app)? {
                     build.add_cmd(build_cmd);
                     build.add_cmd(format!("cp target/{target}/release/{name} bin"));
+=======
```

Comment:
Please resolve this merge artifact

Reply:
Oh nvm it is fixed :)

Reply:
yep sorry that was an accident on my end cherry picking